src/test/java/bio/terra/service/auth/iam/sam/SamIamTest.java

```diff
@@ -691,20 +691,23 @@ void testGetStatus() throws ApiException {
                             {GooglePubSub=class SubsystemStatus {
                                 ok: true
                                 messages: null
+                                additionalProperties: null
                             }}""")));
     }
 
     @Test
     void testGetStatusException() throws ApiException {
-      when(samStatusApi.getSystemStatus()).thenThrow(new ApiException("BOOM!"));
-      assertThat(
-          samIam.samStatus(),
-          is(
-              new RepositoryStatusModelSystems()
-                  .ok(false)
-                  .message(
-                      "Sam status check failed: bio.terra.service.auth.iam.exception.IamInternalServerErrorException: "
-                          + "BOOM!")));
+      when(samStatusApi.getSystemStatus()).thenThrow(new ApiException("BOOM!", 502, null, null));
+      var expected =
+          new RepositoryStatusModelSystems()
+              .ok(false)
+              .message(
+                  "Sam status check failed: bio.terra.service.auth.iam.exception.IamInternalServerErrorException: Message: BOOM!\n"
+                      + "HTTP response code: 502\n"
+                      + "HTTP response body: null\n"
+                      + "HTTP response headers: null");
```

Comment:
I think a text block is slightly better here, if it works
```suggestion
                  """
                      Sam status check failed: bio.terra.service.auth.iam.exception.IamInternalServerErrorException: Message: BOOM!
                      HTTP response code: 502
                      HTTP response body: null
                      HTTP response headers: null""");
```